app/actualize_script.php

```diff
@@ -1,6 +1,6 @@
 <?php
-require(dirname(__FILE__) . '/../constants.php');
-require(LIB_PATH . '/lib_rss.php');	//Includes class autoloader
+require(__DIR__ . '/../constants.php');
+require(__DIR__ . '/../cli/_cli.php');
```

Comment:
`__DIR__` should not be required at all. Something like `require('../cli/_cli.php');` should work in all cases.
Same for the other lines.

Reply:
Nope, I would disagree. There is no chdir() in FreshRSS and therefor the current working path is unspecified and depends on how you call the script. That goes especially for cron jobs.

Reply:
`require(relative_path)` is relative to the calling script, and does not depend on current execution dir http://php.net/manual/function.include.php